setup.py

```diff
@@ -381,7 +381,6 @@ def run(self):
         "Programming Language :: Python :: 3.7",
     ],
     entry_points={"console_scripts": ["feast=feast.cli:cli"]},
-    use_scm_version=use_scm_version,
     setup_requires=[
```

Comment:
Same..same... let's keep upstream files without our modifications. These changes will complicate code pulls from upstream. You can easily lose your changes because most of the time you just update local master with upstream/master. Let's talk about it.